nss-day-cohort-45/tabloidfullstack-giving-gillyflowers
# Nd post by cat

## Changes
1. back-end created get post by cat method in repository and handled it in controller
2. front-end created searchcat component; added it to the posts list
3. slight style of posts list

## Testing
1. fetch/Run/start/build 
2. navigate to post list. see cat drop down on left. switch between categories. check it out
